requirements.txt

```diff
@@ -5,3 +5,4 @@ openwisp-utils[rest]~=0.6
 swapper~=1.1
 markdown>=3.2,<3.3
 celery~=4.4
+django>=3.0,<3.2
```

Comment:
I don't think you will need to add this here.

Reply:
Okay I will do the changes.